### PR TITLE
Minor fixes to debugger_v2 guide.

### PR DESCRIPTION
**Pull request by bmd3k**
Minor fixes to the debugger_v2 guide, including:
* Use consistent code markers for certain symbols.
* Update references to "logits" input to a "probs" input, to match the screenshots and what appears in reality.

### CODE REVIEW

**Review by bileschi — APPROVED** [review]
Thanks!
